sql/moz-fx-data-shared-prod/monitoring_derived/telemetry_missing_columns_v3/query.sql

```diff
@@ -20,7 +20,7 @@ extracted AS (
     DATE(submission_timestamp) = @submission_date
     -- https://cloud.google.com/bigquery/docs/querying-wildcard-tables#filtering_selected_tables_using_table_suffix
     -- exclude pings derived from main schema to save on space, 300GB vs 3TB
-    AND _TABLE_SUFFIX NOT IN ('main_v4', 'saved_session_v4', 'first_shutdown_v4')
+    AND _TABLE_SUFFIX NOT IN ('main_v4', 'saved_session_v4', 'first_shutdown_v4', 'main_remainder_v4')
```

Comment:
this also needs to include saved_session_remainder_v4 and first_shutdown_remainder_v4